scripts/scripts.js

```diff
@@ -1299,20 +1373,18 @@ sampleRUM.drain('convert', (elements, cevent, cvalue) => {
           // send conversion event for each experiment that has been seen by this visitor
           sampleRUM('variant', { source: experiment, target: treatment });
         });
-      // send conversion event
-      sampleRUM('convert', { source: cevent, target: cvalue });
+      // send conversion event      
+      sampleRUM('convert', { source: cevent, target: cvalue }, extraData);      
     } catch (e) {
       // eslint-disable-next-line no-console
       console.log('error reading experiments', e);
     }
   }
 });
 
-sampleRUM.always.on('convert', ({ source, target }) => window.adobeDataLayer && window.adobeDataLayer.push({
-  // TODO: this is just me guessing what the event should be called, same for the payload
-  event: 'Conversion',
-  conversion: {
-    thingClickedOn: source,
-    valueOfAClick: target,
+sampleRUM.always.on('convert', ( data, extraData ) => {
+  console.debug('push to datalayer - convert ', extraData);
+  if(window.digitalData) {
+    window.digitalData.push(extraData);
```

Comment:
Put the actual objects here 😄

Reply:
sounds good. I can replace `extraData` by the actual `Element` and generate the datalayer event objects here.